examples/apps/ai-collab/src/app/presence.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	IPresence,
+	LatestMap,
+	type PresenceStates,
+	type PresenceStatesSchema,
+	// eslint-disable-next-line import/no-internal-modules
+} from "@fluidframework/presence/alpha";
+
+export interface User {
+	photo: string;
+}
+
+const statesSchema = {
+	onlineUsers: LatestMap<{ value: User }, `id-${string}`>(),
+} satisfies PresenceStatesSchema;
+
+export type UserPresence = PresenceStates<typeof statesSchema>;
+
+// Takes a presence object and returns the user presence object that contains the shared object states
+export function buildUserPresence(presence: IPresence): UserPresence {
+	const states = presence.getStates("name:app-client-states", statesSchema);
```

Comment:
A couple questions:
* Where does this name come from? (What is the source of truth for this string?)
* Can it be changed somewhere else, and if so, does this code fail to function as expected at runtime?